homeassistant/components/lektrico/strings.json

```diff
@@ -22,6 +22,38 @@
     }
   },
   "entity": {
+    "binary_sensor": {
+      "state_e_activated": {
+        "name": "State e activated"
+      },
+      "overtemp": {
+        "name": "Overtemp"
+      },
+      "critical_temp": {
+        "name": "Critical temp"
+      },
+      "overcurrent": {
+        "name": "Overcurrent"
+      },
+      "meter_fault": {
+        "name": "Meter fault"
+      },
+      "undervoltage": {
+        "name": "Undervoltage"
+      },
+      "overvoltage": {
+        "name": "Overvoltage"
```

Comment:
I would write `temp` out fully.

Reply:
"Overtemp" is now called "Thermal throttling".
"Critical temp" is called now "Overheating".